src/java.base/unix/classes/sun/nio/fs/UnixFileSystem.java

```diff
@@ -630,6 +633,9 @@ void copyFile(UnixPath source,
             } catch (UnixException x) {
                 if (x.errno() == EEXIST && flags.replaceExisting)
                     throw new FileSystemException(target.toString());
+                if (x.errno() == ENOTDIR) {
+                    x.setError(ENOENT);
+                }
```

Comment:
This is also the target so I don't think this should be changed.

Reply:
Reverted in 0908e75.